packages/react-core/src/demos/examples/Tabs/NestedTabs.tsx

```diff
@@ -108,14 +110,14 @@ export const NestedTabs: React.FunctionComponent = () => {
           <FlexItem flex={{ default: 'flex_1' }}>
             <Card isFullHeight>
               <CardHeader>
-                <Title headingLevel="h2">Title of Card</Title>
+                <Title headingLevel="h3">Title of Card</Title>
```

Comment:
Thank you @mcoker ! Good point, that makes sense. In the current core workspace however these are set to `h3`s. Is this something that would need to also be updated in core for consistency?

Reply:
@jenny-s51 ah good catch! I didn't notice that. Looks like it was created that way when we added the demo in core, though as far as I can tell, there is no requirement that those be different heading levels, and since the size the same, I'd say that implies they're the same level. @mattnolting wdyt? FWIW here's the core PR where that demo was added https://github.com/patternfly/patternfly/pull/3914

Reply:
@mattnolting says this PR is good, opened a core issue to update - https://github.com/patternfly/patternfly/issues/5244